lib/stack/index.js

```diff
@@ -19,6 +19,7 @@ import { Branch } from './branch'
 import { BranchAlias } from './branchAlias'
 import { AuditLog } from './auditlog'
 import { Taxonomy } from './taxonomy'
+import { ManagementToken } from './managementToken/index'
```

Comment:
No need to mention index. The index file serves as root here. Just mention `'./managementToken'`